### PR TITLE
Allow Strings' columns to be resized (#2711)

### LINKED ISSUE #2711: Can't resize columns width in Strings window
**Environment information**
* Operating System: Windows 10 Pro x64
* Cutter version: 2.0.2
* Obtained from:
  - [ ] Built from source
  - [x] Downloaded release from Cutter website or GitHub 
  - [ ] Distribution repository
* File format:

**Describe the bug**

I can't resize the columns of the Strings window. It worked before. When I hover the mouse cursor between to column headings to resize them so that I can read the whole strings column, I don't geht the icon to resize a column. That worked before in 1.9.0.

### PR DESCRIPTION
**Pull request by A0su**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [X] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

I began by comparing the current release of Cutter that I am using to v1.9 as mentioned in the issue to see what the missing functionality looked like. After experimenting with `resizeSections()` and `setSectionResizeMode()` and googling about stretched and interactive columns in Qt I arrive at my current solution.

Before (v1.11): 
before
After: 
after

The only issue I can see is that on startup the columns aren't spaced the same in v11.1 where the Strings column is stretched. Should this also be incorporated into this fix?

Closing issues
Closes #2711

### CODE REVIEW

**Comment by xarkes** [comment]
Hello, thanks for the contribution!
Indeed it would be better if everything is resized to contents at first and to allow resizing as well.
It'd be great if you can find the proper setting to do so :)

**Comment by xarkes** [comment]
What should we do with this? It's indeed nice to be able to resize the columns, however I'm not sure if the current implementation is better than the fixed sized columns we had before.

**Comment by A0su** [comment]
I can try to get the fix out within the next week; sorry have been busy with school lately. To clarify the implementation should present the interface as it does now but also support resizing, correct?

**Comment by xarkes** [comment]
I believe that would be the best, yes!

File: src/widgets/StringsWidget.cpp [hunk 1]
```diff
@@ -214,10 +214,8 @@ StringsWidget::StringsWidget(MainWindow *main)
     });
 
     auto header = ui->stringsTreeView->header();
-    header->setSectionResizeMode(QHeaderView::ResizeMode::ResizeToContents);
```
Comment (whol-hoopa) [inline src/widgets/StringsWidget.cpp:217]: It also works when just changing this line to "header->resizeSections(QHeaderView::ResizeMode::ResizeToContents);"

**Comment by ITAYC0HEN** [comment]
unless specifying maximum-length at start, it can be risky as some strings can be VERY long an the column resizer will be far far to the right.

**Pull request closed without merging**